github/actions/client.go

```diff
@@ -921,6 +921,10 @@ func (c *Client) getActionsServiceAdminConnection(ctx context.Context, rt *regis
 	}
 	defer resp.Body.Close()
 
+	if resp.StatusCode < 200 || resp.StatusCode > 299 {
+		return nil, fmt.Errorf("unexpected response from Actions service during registration call: %v", resp.StatusCode)
```

Comment:
Are we dumping that response somewhere else? We might need to see what the service is returning in order to troubleshoot the root cause of this error